src/Protocol/ActivityPub/Receiver.php

```diff
@@ -1419,18 +1414,7 @@ private static function processObject($object)
 
 		if (in_array($object_data['object_type'], ['as:Audio', 'as:Video'])) {
 			$object_data['alternate-url'] = self::extractAlternateUrl($object['as:url'] ?? []) ?: $object_data['alternate-url'];
-
-			$attachments = self::processAttachmentUrls($object['as:url'] ?? []);
-			foreach ($attachments as $type => $attachment) {
-				$object_data['attachments'][] = [
-					'type' => $type,
-					'mediaType' => $attachment['type'],
```

Comment:
We don't seem to assign the `mediaType` anymore?

Reply:
Indeed, I didn’t check for the exact array key assignments.